From patchwork Thu Dec 27 14:59:08 2018
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: [3/8] c: Delete a stray line in asm inline
X-Patchwork-Submitter: Segher Boessenkool <segher@kernel.crashing.org>
X-Patchwork-Id: 13819
Message-Id: <35014fbfa36ce89dd26fdf43387416a907be4782.1545922222.git.segher@kernel.crashing.org>
To: gcc-patches@gcc.gnu.org
Cc: Segher Boessenkool <segher@kernel.crashing.org>
Date: Thu, 27 Dec 2018 14:59:08 +0000
From: Segher Boessenkool <segher@kernel.crashing.org>
List-Id: <gcc-patches.gcc.gnu.org>

I noticed I accidentally copied a line too many from the "volatile"
handling to the "inline" handling.  This fixes it.

Tested on powerpc64-linux {-m32,-m64}; committing as trivial and obvious.

Segher

2018-12-08  Segher Boessenkool  <segher@kernel.crashing.org>

gcc/c/
	* c-parser (c_parser_asm_statement) [RID_INLINE]: Delete stray line
	setting "quals".
---
 gcc/c/c-parser.c | 1 -
 1 file changed, 1 deletion(-)

-- 
1.8.3.1

diff --git a/gcc/c/c-parser.c b/gcc/c/c-parser.c
index 51cc2be..e45f70b 100644
--- a/gcc/c/c-parser.c
+++ b/gcc/c/c-parser.c
@@ -6321,7 +6321,6 @@ c_parser_asm_statement (c_parser *parser)
 	if (!is_inline)
 	  {
 	    is_inline = true;
-	    quals = c_parser_peek_token (parser)->value;
 	    c_parser_consume_token (parser);
 	  }
 	else
